octobot_services/channel/abstract_service_feed.py

```diff
@@ -0,0 +1,35 @@
+#  Drakkar-Software OctoBot-Notifications
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from abc import ABCMeta
+
+from octobot_channels.channels.channel import Channel
+from octobot_channels.producer import Producer
+from octobot_channels.consumer import Consumer
+
+
+class ServiceFeedChannelConsumer(Consumer):
```

Comment:
Why `ServiceFeedChannelConsumer` and not `AbstractServiceFeedChannelConsumer` ? Because this consumer has no implementation.